server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsUnlock.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.usecases;
+
+import org.eclipse.lsp.cobol.usecases.common.CICSTestUtils;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test CICS UNLOCK command. Documentation link: <a
+ * href="https://www.ibm.com/docs/en/cics-ts/6.x?topic=summary-unlock">UNLOCK
+ * Command</a>
+ *
+ * <p>This class tests all variations of the UNLOCK command found in the link above.
+ */
+public class TestCicsUnlock {
+  private static final String ALL_OPTIONS =
+          "UNLOCK FILE(12) TOKEN(12) SYSID(31)";
+
+  private static final String SOME_OPTIONS_ONE =
+          "UNLOCK TOKEN(31) FILE(12)";
+  private static final String SOME_OPTIONS_TWO =
+          "UNLOCK SYSID(21) FILE(12)";
+
+  private static final String BARE_OPTIONS =
+          "UNLOCK FILE(12)";
+
```

Comment:
Add a test with e.g. just the `SYSID` operand to verify the missing `FILE` operand is correctly diagnosed.